### DISCUSSION TITLE
Project Activity

### DISCUSSION THREAD

**Discussion by garcipat** [bchavez/Bogus#498] [General]
Is this project still active and is there somebody still reviewing Pull Requests?
I have seen that there weren't nearly any contributions since 2021 but a few PRs are open.

Any life sign of the project owner? @bchavez?

**Comment by bchavez** [comment]
Yes, I'm still here. The significant drop in commits and PR reviews directly correlates to getting a full-time job.

**Bogus** has 50+ million downloads and I don't just accept any random pull request.

To keep the support load low; and how I maintain this project:
* Each pull request needs to be thoroughly reviewed and vetted.
* Each pull request needs to align with my maintainability and API design standards and align with an overall vision for cohesiveness.

Once I accept and merge a pull request; I have to maintain that code and that decision indefinitely. I don't have the luxury to call-in the original developer who wrote that original PR to fix a problem. The buck ultimately stops with me. If there's a problem with that new code, I have an upper bound of 50+ million complaints.

I know it's stating the obvious, but it's my open-source project and **Bogus** happens at my pace; _**not at the pace of business.**_ Unless some super big company like **Microsoft** hires me to work on **Bogus** full-time (or if **Bogus Premium** subscriptions ever surpass my full-time salary); then you'll likely see faster commits and PR merges. I wish that would happen 🤞, but we all wish for great things and sometimes they never happen.

Also,
* I do have a life outside of work.
* I do have a family.
* I do have a home.
* I do have a full-time job.

And I am **_very lucky and very thankful_** for those. 🙏 

Only when all of the above life-priorities are taken care of; and when I have free time, and when I'm happy, then and only then will I work on the things I want to work on. That's a basic guideline I try to follow for a happy life.

**Reply by garcipat** [reply]
Thanks for the response. I was just asking and without any sign Its not so obvious.
I understand all that and I had no expectations, I just wanted to get some clarification I now clearly have since I just got involved into GitHub in general.
I hope you get what you deserve, I love Bogus, thats why I'm using it a lot, I even presented it and I intend to write an article inn my company to make it have a bit of a spotlight.

cheers
